### ISSUE TITLE
ReactJS port

### ISSUE THREAD

**Issue by db81** [springweblobby/swl-website#24]
This is a meta issue for things that need to be done before the ReactJS port is usable.
 -  [x] Unitsync: execution strands, bindings to QWeblobbyApplet.
 -  [x] Process launcher.
 -  [x] Startbox drawing and rendering.
 -  [x] Modoptions, mapoptions and AI options.
 -  [x] Downloads.
 -  [x] Map selection.
 -  [x] Game selection.
 -  [x] Multiplayer battles.
 -  [x] Engine options.
 -  [x] Nightwatch support.
 -  [x] Help screen.
 -  [ ] Generic home screen.

**Comment by Anarchid** [comment]
What does 'Generic home screen' stand for?

**Comment by db81** [comment]
That's up to debate. The game-specific menus should probably be removed or at least reworked to be functional. The only useful feature they have is quick creation of SP rooms, it would be good to have something do that in place of the menus.

The layout was made with the menus in mind so without them there's a lot of empty space that could be used.

**Comment by db81** [comment]
Can be closed for now I think, a redesigned home screen should have its own issue.
